src/game/state.ts

```diff
@@ -12,6 +13,25 @@ export const initGameState = (): GameState => ({
   currentPlayer: BLACK_PLAYER,
   isRunning: true,
   winner: null,
+  isDraw: false,
   turnNumber: 0,
   message: "Welcome to Kamon 🍱 ! Black player, you turn",
 });
+
+export const checkIfDraw = (turnNumber: number): boolean => {
+  return turnNumber > 35;
```

Comment:
I believe you should check `if (newGameState.winner != null)` here too

Reply:
fixed condition